packages/rum-core/test/benchmarks/transaction-service.bench.js

```diff
@@ -28,15 +28,12 @@ import { createServiceFactory } from '../../src/index'
 suite('TransactionService', () => {
   const serviceFactory = createServiceFactory()
   const transactionService = serviceFactory.getService('TransactionService')
-  const configService = serviceFactory.getService('ConfigService')
-  // To avoid timeouts interfering with benchmark results
-  configService.setConfig({
-    checkBrowserResponsiveness: false
-  })
 
   benchmark('page-load transaction overhead', () => {
     const tr = transactionService.startTransaction('/index', 'page-load', {
-      managed: true
+      managed: true,
+      // To avoid timeouts interfering with benchmark results
+      checkBrowserResponsiveness: false
```

Comment:
Do we still need this option? This seems to be the only place that uses this option can we do something else here?